assets/scss/variables/_color-pallete.scss

```diff
@@ -0,0 +1,26 @@
+// *** Primary ***
```

Comment:
Please change naming conventions to `kebab-case`. http://thesassway.com/beginner/variable-naming

Reply:
Yeah, one naming convention is good.